packages/react-liveness/src/components/FaceLivenessDetector/service/utils/streamProvider.ts

```diff
@@ -198,13 +198,15 @@ export class LivenessStreamProvider {
       this.videoRecorder.videoStream
     )();
 
+    const mediaSettings = this.stream.getTracks()[0].getSettings();
+
     const response = await this._client.send(
       new StartFaceLivenessSessionCommand({
         ChallengeVersions: 'FaceMovementAndLightChallenge_1.0.0',
         SessionId: this.sessionId,
         LivenessRequestStream: livenessRequestGenerator,
-        VideoWidth: this.videoEl.videoWidth.toString(),
-        VideoHeight: this.videoEl.videoHeight.toString(),
+        VideoWidth: (mediaSettings.width ?? this.videoEl.width).toString(),
+        VideoHeight: (mediaSettings.height ?? this.videoEl.height).toString(),
```

Comment:
Would it make sense to resolve both height/width from one source or does it not matter? e.g
```ts
const getVideoDimensions = (mediaSettings, videoElement) => {
  if (!!mediaSettings.height && !!mediaSettings.width) {
    // return mediaSettings values
  }

  // return videoElement values
}
```

Reply:
Think it doesnt matter since it should usually have both but this is probably a safer change to make, I believe we're going to test this change on alpha first before proceding though